DataAccess/DataAccess.Postgre/PostgreRepositoryManager.hpp

```diff
@@ -13,7 +14,17 @@ namespace DataAccess
 class PostgreRepositoryManager: public IRepositoryManager
 {
 public:
-	explicit PostgreRepositoryManager(const std::shared_ptr<IAdapter>& repositoryContainer)
-		: IRepositoryManager(repositoryContainer) {}
```

Comment:
You could take out the repository registration method and leave the constructor, thus reducing the code from 8 lines to 3.

Reply:
By and large, Data Access is a framework. A framework implies contract programming. In this case, our contract is an agreement to define the `registerRepositories()` method. If you try to transfer these activities to the constructor, then you need to change the structure of the inherited interface.
I hope you understand what the problem is, I will ask about it in one of the next sessions.